manifests/t/TUitTUWien/TUtoolbox/2023.0.0.666/TUitTUWien.TUtoolbox.installer.yaml

```
@@ -0,0 +1,22 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: TUitTUWien.TUtoolbox
+PackageVersion: 2023.0.0.666
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+Installers:
+  - Architecture: neutral
```

```suggestion
  - Architecture: x86
```
Neutral includes all architectures including arm and arm64

---

That's right. And it works tested on them. So need to change.

---

@chbwien We've actually move away from using `Neutral` as an architecture type. Best practice is that if it works on all platforms, then you should create separate Installer nodes for them as such

```yaml
PackageIdentifier: TUitTUWien.TUtoolbox
PackageVersion: 2023.0.0.666
InstallerType: inno
InstallerSwitches:
  Custom: /suppressmsgboxes
MinimumOSVersion: 10.0.0.0
Scope: machine
Platform:
  - Windows.Desktop
UpgradeBehavior: install
Installers:
- Architecture: x64
  InstallerUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/versions/TUToolBox_2023_0_0_666.exe
  InstallerSha256: 64bc1e36cc2d980bab084980cf96581ab6b825f9dbee41954f1179e615b01d18
- Architecture: x86
  InstallerUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/versions/TUToolBox_2023_0_0_666.exe
  InstallerSha256: 64bc1e36cc2d980bab084980cf96581ab6b825f9dbee41954f1179e615b01d18
- Architecture: arm
  InstallerUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/versions/TUToolBox_2023_0_0_666.exe
  InstallerSha256: 64bc1e36cc2d980bab084980cf96581ab6b825f9dbee41954f1179e615b01d18
- Architecture: arm64
  InstallerUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/versions/TUToolBox_2023_0_0_666.exe
  InstallerSha256: 64bc1e36cc2d980bab084980cf96581ab6b825f9dbee41954f1179e615b01d18
ManifestType: installer
ManifestVersion: 1.2.0
```
You can use this as your installer manifest. I've moved the common fields outside as they get inherited in each installer node. Also you don't need to pass Silent and SilentWithProgress switches as they get automatically passed to installer when installer type is set to inno.